web/src/compose.js

```diff
@@ -169,7 +174,27 @@ export function send_message_success(request, data) {
     }
 }
 
-export function send_message(request = create_message_object()) {
+export function toggle_split_messages() {
+    const state = compose_validate.is_split_messages_enabled();
+    compose_validate.set_split_messages_enabled(!state);
+    // preview area and compose banner should be updated with the new setting
+    if (is_preview_area_visible()) {
+        // update the preview area if it was already visible
+        clear_preview_area();
+        show_preview_area();
+    }
+}
+
+export function send_message(
+    message_content = compose_state.message_content(),
+    split_messages = compose_validate.is_split_messages_enabled(),
```

Comment:
This naming is very vague, something like `split_messages_enabled` should be much clearer

Reply:
Done